CONTRIBUTING/README.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
```

Comment:
Would be nice to have this be linked from main README..

Reply:
agree, let's make that a separate PR